src/main/resources/swagger/ga4gh-workflow-execution.yaml

```diff
@@ -0,0 +1,289 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: CWL workflow execution service API
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+paths:
+  /jobs:
+    post:
+      summary: submit a new job
```

Comment:
`job` likely doesn't mean what you intend for it to mean. Please refer to https://github.com/ga4gh/workflow-execution-schemas/blob/master/Definitions.md